src/pages/home/report/EmojiPickerMenu/index.js

```diff
@@ -343,9 +343,12 @@ class EmojiPickerMenu extends Component {
             return;
         }
 
-        const newFilteredEmojiList = _.filter(this.emojis, emoji => (
+        // We're skipping "Frequently Used" emojis to avoid duplicate results. Frequently Usd Emoji is the 0th index category.
+        // Hence, we slice the array from the 1st index category onwards
```

Comment:
IMO, Imperative sentences makes a better comment.
```suggestion
        // Skip "Frequently Used" emojis to avoid duplicate results. 
        // Frequently used emojis are on the 0th index category. Hence, slice the array from the 1st index category onwards.
```